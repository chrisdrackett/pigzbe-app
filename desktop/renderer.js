import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
