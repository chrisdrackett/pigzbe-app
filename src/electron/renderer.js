import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
