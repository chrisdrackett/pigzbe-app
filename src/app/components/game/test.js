import React from 'react';
import renderer from 'react-test-renderer';
import Game from './';

jest.mock('../../../game/game.html', () => 'html');
jest.mock('WebView', () => 'WebView');

describe('Game', () => {
    test('renders correctly', () => {
        const navigation = {
            navigate: jest.fn(),
            addListener: jest.fn()
        };
        const tree = renderer.create(<Game navigation={navigation}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
