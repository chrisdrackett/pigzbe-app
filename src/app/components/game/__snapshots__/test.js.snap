// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game renders correctly 1`] = `
<WebView
  domStorageEnabled={true}
  javaScriptEnabled={true}
  onMessage={[Function]}
  source="html"
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "#003278",
      "flex": 1,
      "justifyContent": "center",
      "paddingTop": 20,
    }
  }
/>
`;
